api/getPageContentByPathname/index.ts

```diff
@@ -0,0 +1,28 @@
+import { CONTENTFUL_API_URL, headers } from '../constants'
+import { HttpRequestMethod } from '../types'
```

Comment:
Ah right that makes sense that we can't import an `enum` explicitly as a type when we use it as a value 😜

Reply:
True dat. I was thinking whether it belongs in the types file, but as it's a TS thing, I'm thinking that that's as good a place as any.